cpo/lib/openshift/types/get_pod_entry.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright 2022 IBM Corporation
```

Comment:
Remove and either return and parse JSON (`oc get pods --output json`) or use JSONPath (`oc get pods {pod_name} --output jsonpath="{.status.phase}"`) [see `cpo/lib/openshift/oc.py`]